src/model/keywords.js

```diff
@@ -15,7 +15,7 @@ const accessiblityKeywords = {
   VALUE: "VALUE",
 };
 
-const outputType = {
+const outputTypes = {
```

Comment:
I'd say the previous name wasn't that bad, and I would rather keep it to make it easier later to read the git changes.

Reply:
For these lines, outType (in global scope) & outType (in local scope) is confusing. That's why changed it to this.
Also if you see other objects in this file, they're plural.

- https://github.com/PicnicSupermarket/localicious/pull/63/files#diff-c944d67b8483c76bf55d7fbb521e2c06bb44c33876b94593b4ce972d7e4c331bR29-R31 & https://github.com/PicnicSupermarket/localicious/pull/63/files#diff-c944d67b8483c76bf55d7fbb521e2c06bb44c33876b94593b4ce972d7e4c331bR68-R70